File: CRM/src/app/shared/eHospital_DongNai_A/zzzBS_KhamTrung.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { zzzBS_KhamTrung } from './zzzBS_KhamTrung.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class zzzBS_KhamTrungService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MA_BAC_SI', 'TEN_BAC_SI', 'NGAY_YL', 'MA_CSKCB', 'TEN_CSKCB', 'MA_TINH', 'TEN_TINH', 'MA_THE', 'HO_TEN', 'XML1_ID', 'MA_LK'];

    List: zzzBS_KhamTrung[] | undefined;
    ListFilter: zzzBS_KhamTrung[] | undefined;
    FormData!: zzzBS_KhamTrung;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "zzzBS_KhamTrung";
    }
}

File: Service_eHospital_DongNai_A/Implement/zzzBS_KhamTrungService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class zzzBS_KhamTrungService : BaseService<zzzBS_KhamTrung, IzzzBS_KhamTrungRepository>
    , IzzzBS_KhamTrungService
    {
    private readonly IzzzBS_KhamTrungRepository _zzzBS_KhamTrungRepository;
    public zzzBS_KhamTrungService(IzzzBS_KhamTrungRepository zzzBS_KhamTrungRepository) : base(zzzBS_KhamTrungRepository)
    {
    _zzzBS_KhamTrungRepository = zzzBS_KhamTrungRepository;
    }
    }
    }

